docs/source/azure/configuration/helmtutorial.rst

```diff
@@ -4,4 +4,194 @@ Tutorial - Helm Deployment
-(AKS) with the provided helm chart and deployment script.
+(AKS) with the provided helm chart and deployment script. This tutorial makes use of the 
+`Azure CLI <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>`_
+and `helm <https://helm.sh/>`_. Make sure that both are installed and you are logged in to your 
+subscription.
+
+1. Create a Resource Group
+--------------------------
+
+This will hold all of the Azure resources created in this tutorial. You could use an existing 
+resource group too.
+
+.. code-block:: bash
+
+    az group create --name c7n-helm-tutorial --location westus2
+
+2. Create a Storage Account
+---------------------------
+
+This storage account will hold all of the blobs and queues used by the container host. You could 
+also use an existing resource here or multiple storage accounts.
+
+We will also create a blob container. One will host the uploaded policies, and the other will store
+the custodian output files of running policies.
+
+.. code-block:: bash
+
+    az storage account create --resource-group c7n-helm-tutorial --name c7nstorage
+
+    account_key=$(az storage account keys list --account-name c7nstorage --query "[0].value" --output tsv)
+    az storage container create --account-name c7nstorage --account-key $account_key --name c7n-helm-policies
+    az storage container create --account-name c7nstorage --account-key $account_key --name c7n-helm-logs
+
+3. Create a Service Principal
+-----------------------------
+
+This service principal will be used as the identity for the container host. You could also use an existing service principal.
+Remember to save the password to use as the client secret.
+
+.. code-block:: bash
+
+    az ad sp create-for-rbac --name c7n-helm 
+
+This service principal will need the proper permissions to interact with the storage account and any other 
+azure resources that the Azure Container Host interacts with. This includes the resources that 
+policies will run against.
+
+The simplest way to grant these permissions is to make the identity a ``Contributor`` on the target 
+subscription and a ``Storage Blob Data Contributor`` and ``Storage Queue Data Contributor`` on the 
+storage account created above.
+
+.. code-block:: bash
+
+    az role assignment create --assignee <Service Principal Client Id> \
+        --role "Contributor" --scope <Target Subscription Resource Id>
+
+    az role assignment create --assignee <Service Principal Client Id> \
+        --role "Storage Blob Data Contributor" --scope <Storage Account Resource Id>
+
+    az role assignment create --assignee <Service Principal Client Id> \
+        --role "Storage Queue Data Contributor" --scope <Storage Account Resource Id>
+
+4. Create an Application Insights Instance
+------------------------------------------
+
+We will point the container host to this Application Insights to gather logs and monitor the host
```

Comment:
"We will create an Application Insights instance to gather the logs and telemetry generated by the running container host"